adding vale instructions to gh (#6395)
segmentio/segment-docs: styleguide.md

# Style guide
When contributing to the docs, follow the rules in this style guide.

> ## Vale Linter

Reviewer: ```suggestion
## Vale linter
```

> To help you follow the style rules in this document, the Segment docs team uses Vale as a linter that will help you find style improvements. To run Vale:

Reviewer: ```suggestion
The Segment docs team uses Vale as a linter, or tool that helps you follow the style rules in our style guide. To run Vale:
```

> 1. Open the Segment docs folder from GH Desktop > Current Repository > segment-docs > Open in Terminal

Reviewer: ```suggestion
1. Navigate to the `segment-docs` folder, and open it in your terminal.
```

> 2. In Visual Studio Code, copy the relative link to the .md file from tree in the sidebar.

Reviewer: ```suggestion
2. In your terminal, copy the relative link to the .md file you'd like to run through Vale.
```

> 3. type 'vale [relative-link]' into the terminal, press enter.

Reviewer: ```suggestion
3. Type 'vale [relative-link]' into your terminal and press enter.
```

> You can also download and install the [Vale extension](https://marketplace.visualstudio.com/items?itemName=chrischinchilla.vale-vscode) to run Vale on demand.

Reviewer: ```suggestion
VSCode users can also download and install the [Vale extension](https://marketplace.visualstudio.com/items?itemName=chrischinchilla.vale-vscode){:target="_blank”} to run Vale on demand.
```
